compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -56,7 +56,9 @@ TR::CodeCacheSymbolContainer * OMR::CodeCacheManager::_symbolContainer = NULL;
 OMR::CodeCacheManager::CodeCacheManager(TR::RawAllocator rawAllocator) :
    _rawAllocator(rawAllocator),
    _initialized(false),
-   _codeCacheFull(false)
+   _codeCacheFull(false),
```

Comment:
If we're keeping track of current usage, do we still need this flag? I would like to avoid having two ways of checking if the code cache is full if possible.

Reply:
My understand is that the current implementation of `setCodeCacheFull` is called only when the allocation or reservation fails. If we combine setting code cache full into this implementation, we’d be checking it on every alloc action when the current usage is updated. Meanwhile this implementation is not called on code cache reservation. It’d be better to keep them separate.

Reply:
I see.